tests/profiling/collector/test_stack.py

```diff
@@ -636,6 +636,8 @@ def _nothing():
                 "switch",
                 "run",
                 "notify",
+                "main_thread_found",
```

Comment:
I wonder if this set could be determined exactly by doing some deterministic profiling 🤔